src/plugins/saved_objects_finder/public/finder/saved_object_finder.tsx

```diff
@@ -307,13 +308,23 @@ export class SavedObjectFinderUi extends React.Component<
           );
 
           const tooltipText = this.props.getTooltipText?.(item);
-
+          const description = !!item.simple.attributes.description && (
+            <EuiText size="xs" color="subdued" className="eui-textTruncate">
+              {item.simple.attributes.description}
+            </EuiText>
+          );
```

Comment:
Thanks for the changes!

I noticed some UI issues when testing:
- descriptions are being partially covered with tags
- and long description are going offscreen.

Would be great to address them.

<img width="1294" alt="Screenshot 2024-11-07 at 21 11 55" src="https://github.com/user-attachments/assets/9e1c5ac5-015b-4bfc-a27e-e7dd3636c6db">

Reply:
Awesome good catch [6b6fe38](https://github.com/elastic/kibana/pull/198816/commits/6b6fe38f8f9eb47c515e42e0ddf04c4c6ec3cfda) for changes